receptorctl/receptorctl/cli.py

```diff
@@ -49,10 +49,13 @@ def get_rc(ctx):
 
 @cli.command(help="Show the status of the Receptor network.")
 @click.pass_context
-def status(ctx):
+@click.option('--json', 'printjson', help="Print as JSON", is_flag=True)
+def status(ctx, printjson):
     rc = get_rc(ctx)
     status = rc.simple_command("status")
-
+    if printjson:
+        print(status)
```

Comment:
is this right? Shouldn't be something like `json.dumps(status)`

Reply:
at this point in the code, status is already JSON

Reply:
I was wrong, it was a python object. calling json.dumps

Reply:
Looks perfect now :)